src/validation-group.js

```diff
@@ -491,7 +491,7 @@ export class ValidationGroup {
    * @param message either a static string or a function that takes two arguments: newValue (the value that has been evaluated) and threshold.
    * @returns {ValidationGroup} returns this ValidationGroup, to enable fluent API
    */
-  withMessage(message) {
+  withMessage(message: any): ValidationGroup {
```

Comment:
don't add `any` as the parameter type if the type can be formally specified - in this case i think following would be better: `string | (newValue: any, threshold: any) => any`